go/pkg/apis/enricher/framework/java/jboss_eap_detector.go

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc.
+ ******************************************************************************/
+package enricher
+
+import (
+	"context"
+	"encoding/xml"
+	"strings"
+
+	"github.com/redhat-developer/alizer/go/pkg/apis/model"
+	utils "github.com/redhat-developer/alizer/go/pkg/utils"
+)
+
+type JBossEAPDetector struct{}
+
+type JBoss_Standalone_Xml struct {
+	SocketBindingGroup struct {
```

Comment:
Why are we looking for socket-binding? The maven project doesn't contain a standalone.xml file. The standalone.xml is produced during maven build (thanks to the eap-maven-plugin maven plugin that provision an EAP server in target/server directory).

Reply:
My understanding here was that `standalone.xml` was a way (perhaps the only) to achieve port recognition for this framework. TBH I haven't found a more efficient solution for this cause.